specification/frontdoor/resource-manager/Microsoft.Network/preview/2018-08-01-preview/frontdoor.json

```diff
@@ -1740,6 +1746,18 @@
       "description": "Routing rules to apply to an endpoint",
       "type": "object",
       "properties": {
+        "routeType": {
+          "type": "string",
+          "enum": [
+            "Forwarding",
```

Comment:
Nit: "Forward" instead of "Forwarding", or else "Redirecting" instead of "Redirect"